plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/OpenShiftExplorerLabelProvider.java

```diff
@@ -204,7 +204,14 @@ public Object visit(ITags capability) {
 				return NLS.bind("({0})", org.apache.commons.lang.StringUtils.join(capability.getTags(), ", "));
 			}
 		}, null);
-		return style(template.getName(), tags);
+		StringBuilder qualifier = new StringBuilder();
+		if(!StringUtils.isEmpty(tags)) {
```

Comment:
Depending upon if this StringUtils is from apache commons, there is an 'isNotEmpty' method